process/elasticproc/transactions/smartContractResultsProcessor.go

```diff
@@ -186,7 +187,7 @@ func (proc *smartContractResultsProcessor) prepareSmartContractResult(
 		Tokens:             converters.TruncateSliceElementsIfExceedsMaxLength(res.Tokens),
 		Receivers:          receiversAddr,
 		ReceiversShardIDs:  res.ReceiversShardID,
-		IsRelayed:          res.IsRelayed,
+		IsRelayed:          false,
```

Comment:
:+1:

Reply:
maybe add a comment for this

Reply:
applied a better backwards compatible solution